source/lambda/apiprocessor/redaction.py

```diff
@@ -33,7 +32,7 @@
 MAXIMUM_NUM_OF_HEADERS_FOOTERS = 3
 
 # we expect a redacted item to have 6 keys: page, top, left, width, height label
-REDACTED_ITEM_KEYS = 6
+REDACTED_ITEM_NUM_KEYS = 5
```

Comment:
Should this be renamed to REQUIRED_REDACTED_ITEM_NUM_KEYS?

Reply:
changed it and defined 2 constants instead with better names